Automattic/wp-calypso
# Help Center: Publish in all locales

#### Proposed Changes

This PR publishes the Help Center to all locales.

#### Testing Instructions



* Checkout branch
* 'yarn dev --sync` from `apps/editing-toolkit`
* Set wordpress in another locale
* Check that translations work




### Screenshot for translation

<img width="422" alt="image" src="https://user-images.githubusercontent.com/17054134/179258128-3f3c9400-276e-4973-9f47-70717a47cdbd.png">
